### PR TITLE
start: link to data step from Pipelines

### LINKED ISSUE #4153: start: get the data needed for the Pipelines page
### Report

Starting in https://dvc.org/doc/start/data-management/data-pipelines and following the instructions is difficult, because it assumes you have the data gotten back in https://dvc.org/doc/start/data-management/data-versioning#expand-to-get-an-example-dataset, specifically:

```console
$ dvc get https://github.com/iterative/dataset-registry \
          get-started/data.xml -o data/data.xml
```

Should we repeat that here, or link to the previous page? @dberenbaum 

> Context: https://discord.com/channels/485586884165107732/1047152728641650709/1048144098768453663

p.s. should we check all pages to see if you can start anywhere? Cc @shcheklein

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Closes #4153

### CODE REVIEW

**Comment by shcheklein** [comment]
Folks, sorry, but I really don't like :(

**Comment by jorgeorpinel** [comment]
Can you repeat your concerns @shcheklein ? I'm not sure I'm understanding the issue.

See https://dvc-org-jorgeorpinel-pa-2qgxqm.herokuapp.com/doc/start/data-management/data-pipelines#pipeline-stages (pretty small change):

image

**Comment by jorgeorpinel** [comment]
p.s. oh I see, from https://github.com/iterative/dvc.org/issues/4153#issuecomment-1335660820

> it's a distraction, and also we don't want this long term

OK so do not change this at all then?

**Comment by dberenbaum** [comment]
What about the `<details>` section above this? Can we drop that and keep this link?

I agree with @jorgeorpinel that it's currently pretty confusing because we give some setup instructions, but they make no sense since they repeat parts of setup from previous sections without doing enough to make this page work as a standalone tutorial.

**Comment by shcheklein** [comment]
> What about the <details> section above this? Can we drop that and keep this link?

hmm ... it won't reproducible? link won't be enough to understand where that code comes from as far as I remember.

> Can you repeat your concerns @shcheklein ? I'm not sure I'm understanding the issue.

Any new note, expandable section etc should have a _very_ strong reason to exist. I think get started is already too complicated (again) and requires some cleanup, not additional explanations. 

We should be trying to do it more explicit and clear by reducing content not making it larger.

Notes btw also signal about some problem. Ideally product and especially docs should be self-explanatory.

(I'm exaggerating everything a bit to make the point about the direction itself)

> OK so do not change this at all then?

My take - yep, if the alternative is to add an additional note.

> I agree with @jorgeorpinel that it's currently pretty confusing because we give some setup instructions, but they make no sense since they repeat parts of setup from previous sections without doing enough to make this page work as a standalone tutorial.

Could you clarify this part please `they repeat parts of setup from previous sections` ?

**Comment by dberenbaum** [comment]
> Could you clarify this part please `they repeat parts of setup from previous sections` ?

My mistake here. I forgot that the code isn't needed for either of the previous sections. 👍 to closing.

File: content/docs/start/data-management/data-pipelines.md [hunk 1]
```diff
@@ -75,6 +75,15 @@ $ dvc stage add -n prepare \
                 python src/prepare.py data/data.xml
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:75]: tmp
```suggestion
                python src/prepare.py data/data.xml 
```

File: content/docs/start/data-management/data-pipelines.md [hunk 2]
```diff
@@ -72,9 +72,18 @@ $ dvc stage add -n prepare \
                 -p prepare.seed,prepare.split \
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:72]: tmp
```suggestion
                -p prepare.seed,prepare.split \ 
```

File: content/docs/start/data-management/data-pipelines.md [hunk 3]
```diff
@@ -69,12 +69,21 @@ Please also add or commit the source code directory with Git at this point.
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:69]: tmp
```suggestion
 
```

File: content/docs/start/data-management/data-pipelines.md [hunk 4]
```diff
@@ -66,15 +66,24 @@ $ pip install -r src/requirements.txt
 Please also add or commit the source code directory with Git at this point.
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:67]:
```suggestion

The data needed to run this example can be found [in a previous page].

[in a previous page]:
  /doc/start/data-management/data-versioning#expand-to-get-an-example-dataset

```

File: content/docs/start/data-management/data-pipelines.md [hunk 5]
```diff
@@ -66,15 +66,24 @@ $ pip install -r src/requirements.txt
...
 
 </details>
-
+
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:69]:
```suggestion

```

File: content/docs/start/data-management/data-pipelines.md [hunk 6]
```diff
@@ -66,15 +66,24 @@ $ pip install -r src/requirements.txt
 Please also add or commit the source code directory with Git at this point.
 
 </details>
-
+ 
 ```cli
 $ dvc stage add -n prepare \
-                -p prepare.seed,prepare.split \
+                -p prepare.seed,prepare.split \
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:72]:
```suggestion
                -p prepare.seed,prepare.split \
```

File: content/docs/start/data-management/data-pipelines.md [hunk 7]
```diff
@@ -66,15 +66,24 @@ $ pip install -r src/requirements.txt
...
-
+ 
 ```cli
 $ dvc stage add -n prepare \
-                -p prepare.seed,prepare.split \
+                -p prepare.seed,prepare.split \ 
                 -d src/prepare.py -d data/data.xml \
                 -o data/prepared \
-                python src/prepare.py data/data.xml
+                python src/prepare.py data/data.xml
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:75]:
```suggestion
                python src/prepare.py data/data.xml
```

File: content/docs/start/data-management/data-pipelines.md [hunk 8]
```diff
@@ -66,15 +66,24 @@ $ pip install -r src/requirements.txt
...
 
+<admon type="info">
+
+The data needed to run this example can be found [in a previous page].
+
+[in a previous page]:
+  /doc/start/data-management/data-versioning#expand-to-get-an-example-dataset
+
+</admon>
+
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:86]:
```suggestion
```

**Review by jorgeorpinel — COMMENTED** [review]
⏳

**Comment by jorgeorpinel** [comment]
> get started is already too complicated (again) and requires some cleanup, not additional explanations

Sure but this PR was meant as a patch so that the current Pipelines page in the GS at least doesn't have a bug while we can get to the next proper iteration of this.

> What about the `<details>` section

For now I hid the note there.

> won't be reproducible? link link won't be enough to understand

Actually getting the data makes it reproducible (currently it isn't). We could repeat the `dvc get` command instead of linking but the link has a better chance of helping readers understand IMO because you realize other work has been done in previous pages.

File: content/docs/start/data-management/data-pipelines.md [hunk 9]
```diff
@@ -65,8 +65,12 @@ $ pip install -r src/requirements.txt
 
 Please also add or commit the source code directory with Git at this point.
 
-</details>
+The data needed to run this example can be found [in a previous page].
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/data-pipelines.md:68]:
```suggestion
<admon type="info:>

The data needed to run this example can be found [in a previous page].

</admon>
```

**Review by shcheklein — APPROVED** [review]

**Pull request merged**